Formula/tesseract.rb

```diff
@@ -21,12 +21,13 @@ class Tesseract < Formula
   end
 
   depends_on "autoconf" => :build
-  depends_on "autoconf-archive" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
   depends_on "pkg-config" => :build
   depends_on "leptonica"
-  depends_on "libtiff"
+  depends_on "libarchive"
+
+  uses_from_macos "curl"
```

Comment:
```suggestion
```
Let's not add this. `uses_from_macos "curl"` isn't for documentation, which is the rationale suggested in the other PR. It's to add a dependency on Homebrew `curl` on Linux. Homebrew itself requires a pre-existing `curl` to function, so we only add `uses_from_macos "curl"` whenever Homebrew's required version of `curl` is too old.

Reply:
OK I have removed it.